aiida/cmdline/commands/cmd_data/cmd_trajectory.py

```diff
@@ -66,16 +65,39 @@ def trajectory_list(raw, past_days, groups, all_users):
 @trajectory.command('show')
 @arguments.DATA(type=types.DataParamType(sub_classes=('aiida.data:core.array.trajectory',)))
 @options.VISUALIZATION_FORMAT(type=click.Choice(VISUALIZATION_FORMATS), default='jmol')
-@show_options
+@options.TRAJECTORY_INDEX()
+@options.WITH_ELEMENTS()
+@click.option(
+    '-c', '--contour', type=click.FLOAT, cls=options.MultipleValueOption, default=None, help='Isovalues to plot'
+)
+@click.option(
+    '--sampling-stepsize',
+    type=click.INT,
+    default=None,
+    help='Sample positions in plot every sampling_stepsize timestep'
+)
+@click.option(
+    '--stepsize',
+    type=click.INT,
+    default=None,
+    help='The stepsize for the trajectory, set it higher to reduce number of points'
+)
+@click.option('--mintime', type=click.INT, default=None, help='The time to plot from')
+@click.option('--maxtime', type=click.INT, default=None, help='The time to plot to')
+@click.option(
+    '--indices', type=click.INT, cls=options.MultipleValueOption, default=None, help='Show only these indices'
+)
+@click.option('--dont-block', 'block', is_flag=True, default=True, help="Don't block interpreter when showing plot.")
 @decorators.with_dbenv()
-def trajectory_show(data, fmt):
+def trajectory_show(data, **kwargs):
     """Visualize a trajectory."""
+    fmt = kwargs.pop('fmt')
```

Comment:
Maybe we can keep `fmt` as positional argument?

Reply:
Thanks, @ryotatomioka - after moving the options back into this file and in a defined order, this makes total sense